src/cryptography/sign.js

```diff
@@ -24,8 +25,35 @@ const signatureHeader = createHeader('SIGNATURE');
 const secondSignatureHeader = createHeader('SECOND SIGNATURE');
 const signatureFooter = createHeader('END LISK SIGNED MESSAGE');
 
-export const signMessageWithPassphrase = (message, passphrase) => {
+const SIGNED_MESSAGE_HEADER = Buffer.from('Lisk Signed Message:\n', 'utf8');
+
+export const writeVarInt = n => {
```

Comment:
This is one of the few occasions where I think a comment would be really helpful. How about linking here? https://en.bitcoin.it/wiki/Protocol_documentation#Variable_length_integer